packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -163,51 +165,66 @@ class TransactionService {
       tr.captureTimings = true
```

Comment:
we can set this flag without checking for route-change or page-load since its managed.